### DISCUSSION TITLE
Issue with connection

### DISCUSSION THREAD

**Discussion by micj1420** [Taxel/PlexTraktSync#1807] [Support] [answered]
Here is the output of the command:  plextraktsync sync

INFO     PlexTraktSync [0.27.11]
INFO     Sync Movie sections: ['Films', 'Autres vidéos à trier', 'Cours de Danse', 'Download', 'Vidéos Personnelles']
INFO     Sync Show sections: ['Series TV']
INFO     OAuth token has expired, refreshing now...
WARNING  HTTPSConnectionPool(host='api-v2launch.trakt.tv', port=443): Max retries exceeded with url: /oauth/token (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert
         handshake failure (_ssl.c:992)'))) for plextraktsync.trakt.TraktApi.me(), retrying after 1 seconds (try: 1/5)
WARNING  HTTPSConnectionPool(host='api-v2launch.trakt.tv', port=443): Max retries exceeded with url: /users/me (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert
         handshake failure (_ssl.c:992)'))) for plextraktsync.trakt.TraktApi.me(), retrying after 2 seconds (try: 2/5)
WARNING  HTTPSConnectionPool(host='api-v2launch.trakt.tv', port=443): Max retries exceeded with url: /users/me (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert
         handshake failure (_ssl.c:992)'))) for plextraktsync.trakt.TraktApi.me(), retrying after 3 seconds (try: 3/5)
WARNING  HTTPSConnectionPool(host='api-v2launch.trakt.tv', port=443): Max retries exceeded with url: /users/me (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert
         handshake failure (_ssl.c:992)'))) for plextraktsync.trakt.TraktApi.me(), retrying after 4 seconds (try: 4/5)
WARNING  HTTPSConnectionPool(host='api-v2launch.trakt.tv', port=443): Max retries exceeded with url: /users/me (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3 alert
         handshake failure (_ssl.c:992)'))) for plextraktsync.trakt.TraktApi.me(), retrying after 5 seconds (try: 5/5)
ERROR    Error: HTTPSConnectionPool(host='api-v2launch.trakt.tv', port=443): Max retries exceeded with url: /users/me (Caused by SSLError(SSLError(1, '[SSL: SSLV3_ALERT_HANDSHAKE_FAILURE] sslv3
         alert handshake failure (_ssl.c:992)')))
ERROR    API didn't respond properly, script will abort now. Please try again later.
ERROR    Last call: plextraktsync.trakt.TraktApi.me((), {})

I renewed the Plex & Trakt tokens without any change. 

It used to work perfectly and suddenly stopped working with always the same message. 

Can you help me?

**Comment by simonc56** [comment, marked as answer]
Read readme file.
Update to last version.

**Comment by micj1420** [comment]
And of course, it solved my issue. However I add to uninstall and reinstall it. An update being not really available.
